packages/api/src/webhooks/cannon.ts

```diff
@@ -158,6 +159,7 @@ export default class WebhookCannon {
       );
     } else {
       console.log("preparing to fire webhook ", webhook.url);
+      const ts = Date.now();
```

Comment:
Since this field is going on the public payload, WDYT of using the full name like `timestamp`? Or maybe `date` like the HTTP header, but I think I prefer `timestamp`

Reply:
Renamed to `timestamp`